### PR TITLE
Add route to change client secret

### PR DESCRIPTION
**Pull request by rien**
There wasn't a way to change the client secret yet. This adds one.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,5 +1,9 @@
 root = true
 
+[*.html]
+indent_style = tab
+indent_size = 4
+
```
Comment (chvp) [inline .editorconfig]: This is redundant, it is already specified below.

File: templates/clients/edit_client.html [hunk 2]
```diff
@@ -2,70 +2,97 @@
...
+
+					<!-- Description -->
+					<div class="field">
+						<label class="label">Description</label>
+				<textarea
+							class="textarea"
+							name="description"
+							rows="4"
+							cols="50"
+							placeholder="Describe what the application does.">{{ client.description -}}</textarea>
```
Comment (chvp) [inline templates/clients/edit_client.html:41]: This indent is weird.

File: templates/clients/edit_client.html [hunk 3]
```diff
@@ -2,70 +2,97 @@
...
+							</label>
+						</div>
+					</div>
+
+					<!-- Redirect URI's -->
+					<div class="field">
+						<label class="label">Redirect URI's</label>
+				<textarea class="textarea" name="redirect_uri_list" rows="4" cols="50" placeholder="http://localhost:3000/auth/callback
+		https://example.com/auth/callback"
+						>{{- client.redirect_uri_list -}}</textarea>
```
Comment (chvp) [inline templates/clients/edit_client.html:67]: This indent is also weird.

**Comment by niknetniko** [comment]
Is this project still active? I would like to use this (pinging @rien @chvp ), so any update on this?

**Review by chvp — APPROVED** [review]

**Comment by niknetniko** [comment]
Thank you @chvp !

**Comment by rien** [comment]
@niknetniko our apologies. Real life took over for a moment.

**Comment by niknetniko** [comment]
@rien thanks for merging! When can we expect a new release? Still waiting :sleeping:

**Pull request merged**